src/tests/suite/daos_base_tx.c

```diff
@@ -704,6 +704,9 @@ dtx_resend_delay(test_arg_t *arg, daos_oclass_id_t oclass)
 	daos_fail_loc_set(0);
 	dtx_set_fail_loc(arg, 0);
 
+	/* Wait for the former delayed RPC before destroying the container to avoid DER_BUSY. */
+	sleep(2);
```

Comment:
can you please explain why 2 sec sleep is needed here? why not 1 or 10?
does it vary if we change some CI hardware?